adapter/src/main/java/com/twilio/kudu/sql/rules/KuduSortRule.java

```diff
@@ -225,6 +232,9 @@ public Boolean visitCall(final RexCall call) {
             return Boolean.FALSE;
           }
         }
+        if (!isDisableInListOptimizationPresent) {
```

Comment:
move to above the for loop since we don't need to do the check if `isDisableInListOptimizationPresent` is false

Reply:
done.